frontend/components/organisms/Question/index.tsx

```diff
@@ -0,0 +1,93 @@
+import Icons from '@/components/atoms/Icons';
+import Avatar from '@/components/molecules/Avatar';
+import Bookmark from '@/components/molecules/Bookmark';
+import Tags from '@/components/molecules/Tags';
+import Votes from '@/components/molecules/Votes';
+import Link from 'next/link';
+import { Fragment } from 'react';
+
+type QuestionProps = {
+  data: {
+    id: number;
+    title: string;
+    content: string;
+    created_at: string;
+    view_count: string;
+    votes: { value: number };
+    tags: { id: number; name: string; is_tag: boolean }[];
```

Comment:
I'll redirect this suggestion to @jescabillas since he created the Question component. I branched out from his branch to add the Answer Component in the Question Detail page.